lib/fluent/supervisor.rb

```diff
@@ -106,6 +107,16 @@ def run_rpc_server
         end
         nil
       }
+      @rpc_server.mount_proc('/api/processes.dump') { |req, res|
+        $log.debug "fluentd RPC got /api/processes.dump request"
+        if Fluent.windows?
+          supervisor_dump_handler_for_windows
+        else
+          Process.kill :CONT, $$
+          send_signal_to_workers(:CONT)
+        end
+        nil
+      }
```

Comment:
It would be better that adding a config param to enable/disable this feature like `@enable_get_dump`, and its default value should be `false (disabled)`.
Or it would be better that we use existing `@enable_get_dump` as is.

Reply:
> Or it would be better that we use existing `@enable_get_dump` as is.

It's not appropriate.
It's designed to use only for `/api/config.getDump` (get config via HTTP), and not used for `/api/config.dump` (dump config to log).

ref: https://www.clear-code.com/blog/2015/10/2.html

Reply:
Oh, I see.
Is there a possibility that we don't have to turn off the API by the option since this doesn't send the content, but just outputs to the local files?

Reply:
I want to remove this API completely for now, I don't think this API is really required.
I think it's okay to add it after someone actually requests it.

Reply:
Since it's for debug usage, I think `fluent-ctl dump` is enough for now.

Reply:
Agreed.
The implementation of the command accomplishes the main purpose of this PR, so I think the API can be removed.

Reply:
I'll do it.

Reply:
Thank you very much!

Reply:
Removed in 4b18c6f34344d455faf75e407927871f1361f6b5